.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Collector Integration Tests
+
+on: 
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
Wouldn't `labeled` make it so this runs whenever we add or remove a label? That seems a little overkill.

Reply:
it would but without it there's seemingly no other way of re-running when a label is added (you could feasibly add a label and then push something to the PR to trigger it, but that doesn't feel like behaviour we want to encourage)

Reply:
But then you could add and remove a bunch of labels and it will trigger just as many jobs, feels like a great way to get DoS'ed.